src/site/content/en/secure/same-site-same-origin/index.md

```diff
@@ -82,33 +85,55 @@ considered "same-origin". Everything else is considered "cross-origin".
 
 ## Site
 
-{% Img src="image/admin/oSRJzCJIr4OjGzUhcNDP.png", alt="Site", width="680", height="142" %}
+{% Aside 'important' %}
+
+The definition of "site" has evolved and it now includes the scheme. This
+effectively changes the definition of "same-site" and "cross-site" as well. What
+has been called "[schemeful same-site](/schemeful-samesite/)" is now "same-site"
+and a new concept of "schemeless same-site" emerged. Read on to learn more.
+
+{% endAside %}
+
+{%
+  Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/GbXtpqme6JcvlxpWRcYf.png",
+  alt="Site (TLD+1)", width="800", height="165"
+%}
 
-Top-level domains (TLDs) such as `.com` and `.org` are listed in the [Root Zone
+[Top-level domains (TLDs)](https://developer.mozilla.org/docs/Glossary/TLD) such
+as `.com` and `.org` are listed in the [Root Zone
 Database](https://www.iana.org/domains/root/db). In the example above, "site" is
-the combination of the TLD and the part of the domain just before it. For
-example, given a URL of `https://www.example.com:443/foo` , the "site" is
-`example.com`.
+the combination of the
+[scheme](https://developer.mozilla.org/docs/Web/HTTP/Basics_of_HTTP/Identifying_resources_on_the_Web#Scheme_or_protocol),
+the [TLD](https://developer.mozilla.org/docs/Glossary/TLD) and the part of the
+domain just before it (We call it TLD+1). For example, given a URL of
+`https://www.example.com:443/foo` , the "site" is `https://example.com`.
+
+### Public Suffix List and eTLD
 
-However, for domains such as `.co.jp` or `.github.io`, just using the TLD of
-`.jp` or `.io` is not granular enough to identify the "site". And there is no
-way to algorithmically determine the level of registrable domains for a
-particular TLD. That's why a list of "effective TLDs"(eTLDs) was created. These
-are defined in the [Public Suffix List](https://wiki.mozilla.org/Public_Suffix_List).
- The list of eTLDs is maintained at
-[publicsuffix.org/list](https://publicsuffix.org/list/).
+For domains that include things such as `.co.jp` or `.github.io`, just using
+`.jp` or `.io` is not granular enough to identify the "site". There is no way to
+algorithmically determine the level of registrable domains for a particular TLD.
+ That's why a list of public suffix that are defined in the [Public Suffix
+List](https://wiki.mozilla.org/Public_Suffix_List) was created. These public
+suffixes are also called **"effective TLDs"(eTLDs)**. The list of eTLDs is
+maintained at [publicsuffix.org/list](https://publicsuffix.org/list/).
 
-The whole site name is known as the eTLD+1. For example, given a URL of
-`https://my-project.github.io` , the eTLD is `.github.io` and the eTLD+1 is
-`my-project.github.io`, which is considered a "site". In other words, the eTLD+1
-is the effective TLD and the part of the domain just before it.
+To identify "site" part of a domain that includes an eTLD, apply the same
```

Comment:
Confused at this phrasing. Identify site as part of a domain that includes an eTLD or identify the "site" part of a domain that includes an eTLD?

Also, either way, I guess the eTLD part is defining the domain as a whole?

Reply:
Latter. For the second question, no. For example, a domain "www.project.github.io", the eTLD is "github.io" and eTLD+1 is "project.github.io" so "www" is not included.